tests/cockpit-tests

```diff
@@ -37,7 +30,7 @@ if sudo -n ip address show dev cockpit1 >/dev/null 2>/dev/null; then
 fi
 
 if [ ! -d cockpit ]; then
-    sudo chown -R user /build
+    sudo chown -R user .
```

Comment:
The script doesn't change directory by itself, so it's not clear that '.' is always the desired location to change. Could this be more explicit?